modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeFooter.tsx

```diff
@@ -22,10 +26,16 @@ interface ObjectDefinitionNodeFooterProps {
 export default function ObjectDefinitionNodeFooter({
 	handleSelectObjectDefinitionNode,
 	isLinkedObjectDefinition,
+	selectedObjectDefinitionId,
 	setShowAllObjectFields,
 	setShowModal,
 	showAllObjectFields,
 }: ObjectDefinitionNodeFooterProps) {
+	const [_, dispatch] = useObjectFolderContext();
+
+	const store = useStore();
```

Comment:
Hey @Artartur, when we create a constant that will be used below, we usually add a white line between the definition and the use, so:

```
	const [_, dispatch] = useObjectFolderContext();
	const store = useStore();
	
	const {edges, nodes} = store.getState();
```
Please find other occurrences and do the same